package.json

```diff
@@ -17,6 +17,7 @@
     "eslint-plugin-jest": "^23.8.2",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-react-hooks": "^3.0.0",
+    "http-proxy-middleware": "^1.0.3",
```

Comment:
Damit definieren wir einen lokalen Proxy (siehe unten in `setupProxy.js`). Dadurch können wir lokal einfach auf `/api` zugreifen, und der API Request wird dann von unserem lokalen Server weitergereicht an den tatsächlichen API Server. Das hat den Vorteil, dass wir keine Cross-Origin Probleme bekommen. Details dazu siehe hier: https://en.wikipedia.org/wiki/Cross-origin_resource_sharing

Damit wird unterbunden, dass wir von `http://localhost:3000` einfach auf `http://xxx-xxxxx.de` gehen können. Damit das doch funktioniert, gehen wir mit Hilfe des Proxies einfach auf `http://localhost:3000/api`, was der Browser erlaubt, und der Proxy dort leitet die Requests dann weiter auf `http://xxx-xxxxx.de`.